content/doc/book/blueocean/index.adoc

```diff
@@ -83,42 +68,28 @@ endif::[]
-increasingly expecting exceptional design. In recent years, developers have
-become more rapidly attracted to tools that are not only functional but are
-designed to fit into their workflow seamlessly and are a joy to use. This shift
-represents a higher standard of design and user experience. Jenkins needs to
-rise to meet this higher standard.
-
-Creating and visualising CD pipelines is something valuable
-for many Jenkins users and this is demonstrated in the 5+ plugins that the
-Jenkins community has created to meet their needs. This indicates a need to
-revisit how Jenkins currently expresses these concepts and consider delivery
-pipelines as a central theme to the Jenkins user experience.
-
-It is not just CD concepts but the tools that developers use
-every day – Github, Bitbucket, Slack, HipChat, Puppet, or Docker. It is about
-more than Jenkins – it is the developer workflow surrounding Jenkins that
-spans multiple tools.
-
-New teams have little time to learn how to assemble their own Jenkins
-experience – they want to improve their time to market by shipping better
-software faster. Assembling that ideal Jenkins experience is something we can
-work together as a community of Jenkins users and contributors to define. As
-time progresses, developers' expectations of good user experience changes and
-the mission of Blue Ocean enables the Jenkins project to respond.
-
-The Jenkins community has poured its sweat and tears into building the most
-technically capable and extensible software automation tool in existence. Not
-doing anything to revolutionize the Jenkins developer experience today is just
-inviting someone else – in closed source – to do it.
+The DevOps world has transitioned from developer tools that are purely functional, to developer tools being part of a "developer experience".
+It is no longer about a single tool, but the many tools developers use throughout the day, and how they work together to achieve a better workflow for the developer.
+
+Developer tool companies like Heroku, Atlassian, and Github have raised the bar for what is considered a good developer experience.
+In recent years, developers have become more attracted to tools that are not only functional, but designed to fit into their existing workflow seamlessly.
+This shift represents a higher standard for design and function, where developers are expecting an exceptional user experience.
+Jenkins needs to rise to meet this higher standard.
+
+Creating and visualising continuous delivery Pipelines is something valuable for many Jenkins users.
+This is demonstrated in the plugins that the Jenkins community has created to meet their needs.
+This also indicates a need to revisit how Jenkins currently expresses these concepts, and consider delivery pipelines as a central theme to the Jenkins user experience.
+
+It is not just continuious delivery concepts, but the tools that developers use every day such as Github, Bitbucket, Slack, HipChat, Puppet, or Docker.
```

Comment:
Based on the [Wikipedia article](https://en.wikipedia.org/wiki/HipChat), let's not mention HipChat.
```suggestion
It is not just continuous delivery concepts, but the tools that developers use every day such as Github, Bitbucket, Slack, Puppet, or Docker.
```

Reply:
Good to know, that makes much more sense!